tests/std/tests/P0896R4_counted_iterator/test.cpp

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <concepts>
+#include <iterator>
+
+#include <range_algorithm_support.hpp>
+using namespace std;
+
+struct instantiator {
+    template <input_or_output_iterator Iter>
+    static constexpr void call() {
+        if constexpr (is_permissive) { // TRANSITION, FIXME (File an issue to investigate this later?)

```

Comment:
We can't merge a FIXME comment and we shouldn't merge a constraint recursion hack like this. I'm still investigating, but it's likely this library component simply won't be usable in permissive mode and we should add a `strict_concepts_matrix.lst` for the test that has no permissive MSVC configurations.

Reply:
Defere this for further analysis. If it takes longer we might mark this PR as blocked?